parse/parser.go.y

```diff
@@ -396,6 +396,9 @@ prefixexp:
             $$ = $1
         } |
         '(' expr ')' {
+            if ex, ok := $2.(*ast.Comma3Expr); ok {
+                ex.AdjustRet = true
+            }
```

Comment:
Just curious: can we get rid of `afunctioncall` above and set `AdjustRet` in one place (here)?

Maybe it is not worth to touch the grammar too much, because the testing is not thorough, so it is just due to an interest. I don't propose to change the code.

Reply:
I tried this, but has no luck to get it work.

Reply:
Okay, thanks!